### ISSUE TITLE
Linking with streaming audio

### ISSUE THREAD

**Issue by mhmmdmz** [alankarmisra/SwiftSiriWaveformView#9]
Request: Can you show some examples of how one could link this to a streaming audio using avplayer? I believe this has to be done using MTAudioProcessingTap. 

Any leads would be a great help.

**Comment by dispatchswift** [comment]
@mhmmdmz Any lead on that?

**Comment by mhmmdmz** [comment]
Nope. The issue is with the metrics available in AVPlayer; they aren't as extensive as AVAudioPlayer and hence I don't think there's a way to do it just with AVPlayer. The other problem is AVAudioPlayer doesn't stream! :( *sob* *sob*
